Breakout/src/game_clock.cpp

```diff
@@ -0,0 +1,51 @@
+#include "game_clock.h"
+
+GameClock::GameClock(float fps = 60.f) : m_fps{ fps }
+{
+	m_frametime = 1.f / m_fps;
+	initiateClock();
```

Comment:
You're calling the initialization method in `Game::run()` again, so the call is actually kind of redundant right now.

Reply:
I agree. 
I guess the reason I did it this way was a lazy way to avoid an uninitialized pointer. There are surely better ways to handle this.